src/widgets/layouts/main/ui/MainLayout/MainLayout.tsx

```diff
@@ -1,16 +1,16 @@
-import { ReactNode } from 'react';
+import { PropsWithChildren } from 'react';
```

Comment:
check this attention, my be lint dont sort import.
try write npm run lint:fix in terminal